python/lsst/meas/base/noiseReplacer.py

```diff
@@ -150,7 +150,7 @@ def __init__(self, config, exposure, footprints, noiseImage=None, exposureId=Non
         # Create in the dict heavies = {id:heavyfootprint}
         for id, fp in footprints.items():
             if fp[1].isHeavy():
-                self.heavies[id] = afwDet.cast_HeavyFootprintF(fp[1])
+                self.heavies[id] = fp[1]
```

Comment:
I love seeing our code become more Pythonic even when only making the required changes :)